public/gift-cards-page/index.ca.html

```diff
@@ -52,7 +52,7 @@ <h2 class='h300 primary50 -font-semi-bold pb4 lg-col-6 md-col-8 mx-auto'>Les tar
       <div class='md-flex items-center'>
         <div class='md-col-5 pb4 md-pb0'>
           <h1 class='h600 -font-extra-bold neutralsolid900 pb4'>Ens encarreguem de tot.</h1>
-          <p class='neutralsolid700'>Només cal que proporcioneu una llista de correus electrònics i us enviarem els codis de bescanvi. També podeu afegir un missatge personal. Només cobrarem el mètode de pagament triat quan s’utilitzi una targeta de regal per fer una donació. Podeu restringir les targetes de regal a determinats tipus de col·lectius (per etiqueta o per amfitrió) així controleu els tipus de col·lectius a qui es dona suportar amb els vostres diners. Cada mes rebràs una factura consolidada per amfitrió.</p>
+          <p class='neutralsolid700'>You just need to provide a list of emails and we’ll send out the redeem codes. You can add a personal message, too. We will only charge your payment method when a gift card is used to make a donation. You can restrict the gift cards to certain types of collectives (by tag, or host) so that you remain in control of the types of collectives that can be supported with your money. Every month you will receive a consolidated invoice per host.</p>
```

Comment:
Sorry, looking to add a new FAQ item, and when trying to see how to add it appropriately with translated pages being present, realized that this had "reverting" to English versions.

Was this intentional? If not, can I help in bringing them back as I make my addition, or should I leave that for a separate PR.